source/gui/settingsDialogs.py

```diff
@@ -1417,130 +1427,106 @@ class BrailleSettingsDialog(SettingsDialog):
 		self.cursorShapes = [s[0] for s in braille.CURSOR_SHAPES]
-		self.shapeList = wx.Choice(self, wx.ID_ANY, choices=[s[1] for s in braille.CURSOR_SHAPES])
+		cursorShapeChoices = [s[1] for s in braille.CURSOR_SHAPES]
+		self.shapeList = sHelper.addLabeledControl(cursorShapeLabelText, wx.Choice, choices=cursorShapeChoices)
 		try:
 			selection = self.cursorShapes.index(config.conf["braille"]["cursorShape"])
 			self.shapeList.SetSelection(selection)
 		except:
 			pass
 		if not self.showCursorCheckBox.GetValue():
 			self.shapeList.Disable()
-		sizer.Add(label)
-		sizer.Add(self.shapeList)
-		settingsSizer.Add(sizer, border=10, flag=wx.BOTTOM)
 
-		sizer = wx.BoxSizer(wx.HORIZONTAL)
 		# Translators: The label for a setting in braille settings to change how long a message stays on the braille display (in seconds).
-		label = wx.StaticText(self, wx.ID_ANY, label=_("Message timeout (sec)"))
-		sizer.Add(label)
-		self.messageTimeoutEdit = wx.TextCtrl(self, wx.ID_ANY)
+		messageTimeoutText = _("Message timeout (sec)")
+		self.messageTimeoutEdit = sHelper.addLabeledControl(messageTimeoutText, wx.TextCtrl)
 		self.messageTimeoutEdit.SetValue(str(config.conf["braille"]["messageTimeout"]))
-		sizer.Add(self.messageTimeoutEdit)
-		settingsSizer.Add(sizer, border=10, flag=wx.BOTTOM)
 
-		sizer = wx.BoxSizer(wx.HORIZONTAL)
 		# Translators: The label for a setting in braille settings to set whether braille should be tethered to focus or review cursor.
-		label = wx.StaticText(self, wx.ID_ANY, label=_("Braille tethered to:"))
+		tetherListText = _("Braille tethered to:")
+		# Translators: The value for a setting in the braille settings, to set whether braille should be tethered to focus or review cursor.
 		self.tetherValues=[("focus",_("focus")),("review",_("review"))]
-		self.tetherList = wx.Choice(self, wx.ID_ANY, choices=[x[1] for x in self.tetherValues])
+		tetherChoices = [x[1] for x in self.tetherValues]
+		self.tetherList = sHelper.addLabeledControl(tetherListText, wx.Choice, choices=tetherChoices)
 		tetherConfig=braille.handler.tether
 		selection = (x for x,y in enumerate(self.tetherValues) if y[0]==tetherConfig).next()  
 		try:
 			self.tetherList.SetSelection(selection)
 		except:
 			pass
-		sizer.Add(label)
-		sizer.Add(self.tetherList)
-		settingsSizer.Add(sizer, border=10, flag=wx.BOTTOM)
 
 		# Translators: The label for a setting in braille settings to read by paragraph (if it is checked, the commands to move the display by lines moves the display by paragraphs instead).
-		item = self.readByParagraphCheckBox = wx.CheckBox(self, label=_("Read by &paragraph"))
-		item.Value = config.conf["braille"]["readByParagraph"]
-		settingsSizer.Add(item, border=10, flag=wx.BOTTOM)
+		readByParagraphText = _("Read by &paragraph")
+		self.readByParagraphCheckBox = sHelper.addItem(wx.CheckBox(self, label=readByParagraphText))
+		self.readByParagraphCheckBox.Value = config.conf["braille"]["readByParagraph"]

```

Comment:
I'm totally cool with just accepting these as stylistic differences, but I'm curious nevertheless:

1. You've changed most places to assign label text to a variable, rather than just specifying the label in the call. Why? I guess this makes the lines shorter, but you could wrap to a new line. This approach seems more "indirect" to me. It also creates a lot more local variables, but that's inconsequential; premature optimisation and all. :)
2. Here (and in quite a few other places), I had a variable like `item` which I just assign temporarily. The reason is largely to make the code less verbose and thus easier to type. You've changed these to either use a new local or, if there was an attribute on `self`, to use that. Why? Was the use of `item` confusing to you; i.e. did you feel you weren't sure which control I was referring to despite the grouping of the code? One other reason I did this was to avoid hash lookups on `self`, but again, inconsequential; premature optimisation and all. :)

Again, not asking for changes here; just curious on your reasoning.